### ISSUE TITLE
Add a build from source instruction

### ISSUE THREAD

**Issue by valignatev** [seanpr96/DebugMod#8]
Hey Sean, thanks a lot for you work! 

Can we hope for a small build-from-source instruction? From what I see, the project references some dlls by the relative path:
https://github.com/seanpr96/DebugMod/blob/master/Source/DebugMod.csproj#L92-L126

It's not very clear what versions are needed and where to obtain them but they are seem to be required to build the project.

**Comment by valignatev** [comment]
I just had a big brain moment and realized that these dlls most likely should come from the Hollow Knight's hollow_knight_data/Managed directory. I'll check it out and close the ticket if that's the case

**Comment by valignatev** [comment]
So, the assemblies from hollow_knight_data directory are correctly recognized, but visual studio can't find system assemblies for .net 3.5 for some reason. Didn't figure it out yet
![image](https://user-images.githubusercontent.com/7038954/78723827-d65e8380-7934-11ea-8668-e604c12f8fc6.png)

**Comment by seresharp** [comment]
https://download.microsoft.com/download/7/0/3/703455ee-a747-4cc8-bd3e-98a615c3aedb/dotNetFx35setup.exe

**Comment by valignatev** [comment]
That didn't help. Looks like it had the same effect as turning .net 3.5 on in windows features, the same dlls are missing. I can find them manually in various subfolders in `C:\Windows\Microsoft.NET\Framework` though. Like, System, System.Data, System.Drawing and System.Xml are in `C:\Windows\Microsoft.NET\Framework\v2.0.50727`. I'm not sure if visual studio should find them automatically or if I can somehow point to them manually.

Just for the full picture: I'm on Windows10, I have Microsoft Visual Studio Community 2019
Version 16.5.3. Somebody told me that it's not feasible to develop for 3.5 in newer VS versions, so please tell me if I have to install something older

**Comment by seresharp** [comment]
You need to enable .NET 3.5 dev tools through the installer as well. VS 2019 does work for 3.5 development

**Comment by valignatev** [comment]
I did all of that I think. This:
![image](https://user-images.githubusercontent.com/7038954/78939490-ee610f00-7abc-11ea-86b6-bee076510fa2.png)
And this:
![image](https://user-images.githubusercontent.com/7038954/78939529-fde05800-7abc-11ea-8e11-b6b23abcc35a.png)
Are these two steps should be enough for this to work?

**Comment by valignatev** [comment]
The same assemblies are missing when I'm trying to create a new dll project that targets .NET framework, so something might be borked within my Windows. The difference is that the new project calls "dependencies" "references".

**Comment by valignatev** [comment]
Ok, after doing some research and asking around, I've found out that it's a known issue where new tooling messed up paths to legacy assembly files: https://github.com/microsoft/msbuild/issues/1333

The issue has multiple possible workarounds, I've chosen to add `FrameworkPathOverride` to my csproj. I don't know how universal this solution is, but it works for my case. The full snippet is:

```xml
<PropertyGroup>
  <FrameworkPathOverride Condition="'$(TargetFramework)' == 'net35'">$(MSBuildProgramFiles32)\Reference Assemblies\Microsoft\Framework\.NETFramework\v3.5\Profile\Client</FrameworkPathOverride>
</PropertyGroup>
```

I don't know if it makes sense to add it to the cproj in the repo, but it works for me. Closing the ticket.
